package.json

```diff
@@ -6,6 +6,7 @@
   "author": "nerdy@bethink.pl",
   "main": "./dist/design-system.umd.js",
   "typings": "./dist/index.d.ts",
+  "license": "UNLICENSED",
```

Comment:
did you contact Adam, to agree on license?

Reply:
`UNLICENSED` is ok.